vowpalwabbit/search_dep_parser.cc

```diff
@@ -93,9 +93,24 @@ void initialize(Search::search &sch, size_t & /*num_actions*/, options_i &option
   all.pairs.swap(newpairs);
   all.triples.swap(newtriples);
 
+  // std::vector<uint8_t> temp_var(pair[0], pair[0]+1);
+  std::vector<std::vector<uint8_t>> newpairs_t(19);// = new std::vector<std::vector<uint8_t>>(20);
```

Comment:
Is this test code that was left in?

Reply:
Yeah, was running into segfault cause of that extra element